app/web/pages/_document.tsx

```diff
@@ -8,8 +8,9 @@ export default class MyDocument extends Document {
     return (
       <Html lang={this.props.locale ?? "en"}>
         <Head>
-          {/* PWA primary color */}
           <meta name="theme-color" content={theme.palette.primary.main} />
+          <link rel="manifest" href="/manifest.json" />
```

Comment:
just this? LGTM

Reply:
Yep! Seems we already had the manifest.json in the file, just wasn't linked.